sql/hive/src/main/scala/org/apache/spark/sql/hive/TableReader.scala

```diff
@@ -240,6 +240,8 @@ class HadoopTableReader(
       fillPartitionKeys(partValues, mutableRow)
 
       val tableProperties = tableDesc.getProperties
+      val avroSchemaEvolutionProperties = Seq(AvroTableProperties.SCHEMA_LITERAL,
```

Comment:
`avroSchemaEvolutionProperties` -> `avroSchemaProperties` because the schema is identical in the most cases.